<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>New</class>
 <widget class="QDialog" name="New">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>666</width>
    <height>360</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: qlineargradient(spread:pad, x1:1, y1:1, x2:0, y2:0, stop:0 rgba(81, 0, 135, 255), stop:0.427447 rgba(41, 61, 132, 235), stop:1 rgba(155, 79, 165, 255));</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="3" column="0" rowspan="2">
    <widget class="QListWidget" name="listWidget">
     <property name="styleSheet">
      <string notr="true"> border: 1px solid rgb(60, 60, 60);
    border-radius: 10px
</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0" rowspan="2">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QCalendarWidget" name="calendarWidget">
       <property name="styleSheet">
        <string notr="true">background-color: black;
color:white</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="1">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLineEdit" name="lineEdit">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="addbtn">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: gray;
font-family: &quot;Segoe UI&quot;, Tahoma, Geneva, Verdana, sans-serif;
border:1px solid;
border-radius: 10px;</string>
       </property>
       <property name="text">
        <string>Add Event</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="1" rowspan="3">
    <layout class="QGridLayout" name="gridLayout" rowstretch="1,4">
     <item row="0" column="0">
      <widget class="QLabel" name="datelabel">
       <property name="styleSheet">
        <string notr="true"> border: 1px solid rgb(60, 60, 60);
    border-radius: 10px
</string>
       </property>
       <property name="text">
        <string>Your Date</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="eventlabel">
       <property name="styleSheet">
        <string notr="true"> border: 1px solid rgb(60, 60, 60);
    border-radius: 10px</string>
       </property>
       <property name="text">
        <string>Your Event</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="hellous">
     <property name="font">
      <font>
       <family>Swis721 BlkEx BT</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"> border: 1px solid rgb(60, 60, 60);
  border-radius: 10px</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
